### PR TITLE
<TableActionCell/> - Add `tooltipProps` to `secondaryActions`

### PR DESCRIPTION
**Pull request by splintor**
### 🔦 Summary
After https://github.com/wix/wix-style-react/pull/5747 was released, we realize we can't control the max width of the displayed tooltip, to adhere to our UX ([WOS3-1366](https://jira.wixpress.com/browse/WOS3-1366))

### ✅ Checklist

- 📚 Change is documented
  - [X] Story
  - [X] API description
- 🔬 Change is tested
  - [X] Component props types
  - [ ] - Visual tests are broken, as disabled example was changed

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5759/

**Comment by splintor** [comment]
@EgoziE Great idea - done! (+ small refactoring of the tooltip unit tests)

File: src/TableActionCell/TableActionCell.js [hunk 1]
```diff
@@ -193,6 +204,7 @@ TableActionCell.propTypes = {
       disabled: PropTypes.bool,
       dataHook: PropTypes.string,
       disabledDescription: PropTypes.string,
+      tooltipProps: PropTypes.shape(TooltipCommonProps),
```
[the remark this answers is not in the document]
Comment (splintor) [reply src/TableActionCell/TableActionCell.js:207]: 👍

File: src/TableActionCell/test/TableActionCell.spec.js [hunk 2]
```diff
@@ -233,12 +233,10 @@ describe('Table Action Cell', () => {
 
     describe('when a secondary action is disabled', () => {
       it('should mark the a visible secondary actions as disabled', async () => {
-        const actionTrigger = jest.fn();
-
         const disabledAction = {
           text: `Disabled Action`,
           icon: <span>Icon</span>,
-          onClick: actionTrigger,
+          onClick: () => void 0,
```
[the remark this answers is not in the document]
Comment (splintor) [reply src/TableActionCell/test/TableActionCell.spec.js]: Done, although I must say I'm not sure why this is needed. Once we assert that the button is disabled, why do we additionally have to assert that clicking it doesn't call its handler? Shouldn't this be tested in `Button`'s component tests?

[the remark this answers is not in the document]
Comment (EgoziE) [reply src/TableActionCell/test/TableActionCell.spec.js]: You're probably right, but I when it comes to old components like this one, I prefer to be on the safe side.

**Pull request merged**